lua/entities/gmod_tardis/shared.lua

```diff
@@ -2,8 +2,8 @@
 
 ENT.Base="gmod_door_exterior"
 ENT.Spawnable=true
-ENT.PrintName="TARDIS Rewrite"
-ENT.Category="Doctor Who"
+ENT.PrintName=" TARDIS " -- spaces are for it to go first in the list
```

Comment:
Can we undo this change and the line below, when the time comes we will rename legacy to TARDIS Legacy but they should still be in the Doctor Who category

Reply:
```suggestion
ENT.PrintName="TARDIS"
```